docs/decisions/0005-cookiecutter-default-branch.rst

```diff
@@ -0,0 +1,25 @@
+Cookiecutter default branch
+===========================
+
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+Github now sets default branch of each new repository to main. And the software industry as a whole has been moving towards default branch name flexibility and away from naming the default branch "master".
+
+There are number of places in edx-cookiecutters where we make assumption about the name of the default branch. Until now, we assumed that name was "master".
+
+Decision
+--------
+
+edx-cookiecutters tooling will assuming new repositories created using the cookiecutters will have the default name of "main".
+
+Consequences
+------------
+
+Lots of tooling in Open edX ecosystem assumes the default branch has the name "master". This decision might result in necessary changes to our tooling to accommodate multiple default branch names.
```

Comment:
Potential additions/updates:
1.
```
Using "main" for a new repo for a library should be safe at this time, but using it for a new IDA may required additional changes to support it.
```
2.
```
Although much of our tooling may need to be updated to handle both "main" and "master" for some transitional period, this decision is more opinionated, and continuing to use "master" for a new repository would require manual changes after using the cookie-cutter.
```

Reply:
I incorporated these into the consequences. Though I moved things around a bit.